<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BhosaleA" failures="0" tests="3" name="amazon_Project1.Project1_Amazon.Testcase01_Registration" time="142.153" errors="1" timestamp="2025-01-30T20:28:41 IST" skipped="1">
  <testcase classname="amazon_Project1.Project1_Amazon.Testcase01_Registration" name="registration" time="63.870">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.getClass()&quot; because &quot;exceptionDriver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.getClass()" because "exceptionDriver" is null
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:85)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:228)
at amazon_Project1.Project1_Amazon.RegistrationPage_Project1.starthere_Register(RegistrationPage_Project1.java:35)
at amazon_Project1.Project1_Amazon.Testcase01_Registration.registration(Testcase01_Registration.java:9)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1575)
]]>
    </error>
    <system-out>
      <![CDATA[result is failed for test=null]]>
    </system-out>
  </testcase> <!-- registration -->
  <testcase classname="amazon_Project1.Project1_Amazon.Testcase01_Registration" name="registration" time="67.623">
    <skipped/>
  </testcase> <!-- registration -->
  <testcase classname="amazon_Project1.Project1_Amazon.Testcase01_Registration" name="registration" time="10.660">
    <system-out>
      <![CDATA[result is passed for test=registration]]>
    </system-out>
  </testcase> <!-- registration -->
  <system-out>
    <![CDATA[result is passed for test=addandremove
result is passed for test=addandremove
result is passed for test=registration
result is passed for test=addtocart
result is failed for test=null
result is passed for test=editprofile
result is failed for test=null
result is passed for test=searchproduct1
result is passed for test=verifyFilter
result is failed for test=null
result is passed for test=cartpage_withoutlogin
result is passed for test=verifySort
result is passed for test=applycoupen
result is passed for test=editprofile
result is passed for test=loginwithcorrectcred
result is passed for test=loginwithcorrectcred
result is passed for test=searchproduct1
result is passed for test=verifyProductDetails
result is passed for test=verifyFilter
result is failed for test=null
result is failed for test=null
result is failed for test=null
result is passed for test=loginwithincorrectcred
result is passed for test=loginwithcorrectcred
result is passed for test=loginwithcorrectcred
result is passed for test=allpaymentmethod
result is failed for test=null
result is failed for test=null
result is passed for test=loginwithincorrectcred
result is failed for test=null
result is failed for test=null
result is failed for test=null]]>
  </system-out>
</testsuite> <!-- amazon_Project1.Project1_Amazon.Testcase01_Registration -->
